desktop/ui/src/main/java/org/datacleaner/panels/result/AnalyzerResultPanel.java

```diff
@@ -99,6 +100,7 @@ public AnalyzerResultPanel(RendererFactory rendererFactory, ProgressInformationP
     public void setResult(final AnalyzerResult result) {
         final String headerText = getHeaderText();
         _progressInformationPanel.addUserLog("Rendering result for " + headerText);
+        _headerLabel.setText(LabelUtils.getLabel(_componentJob, result, false, false, false));
```

Comment:
I guess you don't need to move this logic down here - the ComponentJob is also available in the constructor, so why not just do it immediately and not change the label later?